### PR TITLE
Feature/raise brain err

### PR DESCRIPTION
**Pull request by fraibacas**

### CODE REVIEW

File: Products/Zuul/catalog/model_catalog.py [hunk 1]
```diff
@@ -111,12 +111,13 @@ def getObject(self):
...
         try:
             obj = parent.unrestrictedTraverse(self.getPath())
-        except (NotFound, KeyError, AttributeError):
+        except (NotFound, KeyError, AttributeError) as e:
             msg = "Unable to get object from brain. Path: {0}. Model catalog may be out of sync. "
             msg += "Will attempt to delete the object from model catalog."
             log.error(msg.format(self.uid))
             # unindex uid
             getUtility(IModelCatalog).get_client(self).unindex_uid(self.uid)
+            raise e
```
Comment (iancmcc) [inline Products/Zuul/catalog/model_catalog.py]: nah bro, ignore `as e` and do:
```
info = sys.exc_info()
raise info[0], info[1], info[2]
```
Pretty sure that's still timely information

File: Products/Zuul/catalog/tests/testModelCatalog.py [hunk 2]
```diff
@@ -43,8 +43,13 @@ def test_stale_brain(self):
         self.assertTrue( results.total == 1 )
         brain = results.results.next()
         self.assertTrue(brain.getPath() == bad_uid)
-        obj = brain.getObject()
-        self.assertIsNone(obj)
+        # Ensure getObject raises an exception
+        expception_raised = False
+        try:
+            obj = brain.getObject()
+        except:
```
Comment (iancmcc) [inline Products/Zuul/catalog/tests/testModelCatalog.py]: `except Exception`, please. Otherwise you'll capture interrupts.

Comment (iancmcc) [inline Products/Zuul/catalog/tests/testModelCatalog.py]: although this is a test, so I guess who cares

File: Products/Zuul/catalog/model_catalog.py (hunk 1 above)
Comment (fraibacas) [inline Products/Zuul/catalog/model_catalog.py]: isn't that the same as `raise` with no parameters?

**Comment by fraibacas** [comment]
Is sys imported in model_catalog.py?

File: Products/Zuul/catalog/model_catalog.py [hunk 3]
```diff
@@ -112,11 +112,13 @@ def getObject(self):
         try:
             obj = parent.unrestrictedTraverse(self.getPath())
         except (NotFound, KeyError, AttributeError):
+            info = sys.exc_info()
```
Comment (fraibacas) [inline Products/Zuul/catalog/model_catalog.py]: Is sys imported?

**Comment by tgjonestx** [comment]
retest this

**Comment by tgjonestx** [comment]
retest this

**Pull request merged**